### ISSUE TITLE
qwen-image-edit-2509，相同输入，使用diffsynth-studio与使用diffusers的输出不同

### ISSUE THREAD

**Issue by yangguoquan001** [modelscope/DiffSynth-Studio#966]
diffsynth-studio代码：
`import torch
from PIL import Image
from diffsynth.pipelines.qwen_image import QwenImagePipeline, ModelConfig

pipe = QwenImagePipeline.from_pretrained(
    torch_dtype=torch.bfloat16,
    device="cuda",
    model_configs=[
        ModelConfig(path=[
            "autodl-tmp/qwen-image-edit-2509/transformer/diffusion_pytorch_model-00001-of-00005.safetensors",
            "autodl-tmp/qwen-image-edit-2509/transformer/diffusion_pytorch_model-00002-of-00005.safetensors",
            "autodl-tmp/qwen-image-edit-2509/transformer/diffusion_pytorch_model-00003-of-00005.safetensors",
            "autodl-tmp/qwen-image-edit-2509/transformer/diffusion_pytorch_model-00004-of-00005.safetensors",
            "autodl-tmp/qwen-image-edit-2509/transformer/diffusion_pytorch_model-00005-of-00005.safetensors",
        ], offload_device="cpu"),
        ModelConfig(path=[
            "autodl-tmp/qwen-image-edit-2509/text_encoder/model-00001-of-00004.safetensors",
            "autodl-tmp/qwen-image-edit-2509/text_encoder/model-00002-of-00004.safetensors",
            "autodl-tmp/qwen-image-edit-2509/text_encoder/model-00003-of-00004.safetensors",
            "autodl-tmp/qwen-image-edit-2509/text_encoder/model-00004-of-00004.safetensors",
        ], offload_device="cpu"),
        ModelConfig(path="autodl-tmp/qwen-image-edit-2509/vae/diffusion_pytorch_model.safetensors", offload_device="cpu"),
    ],
    tokenizer_config=ModelConfig(path="autodl-tmp/qwen-image-edit-2509/tokenizer/"),
    processor_config=ModelConfig(path="autodl-tmp/qwen-image-edit-2509/processor/"),
)
pipe.enable_vram_management()

img = Image.open(f"test_{i}.png")
width, height = img.size
if img.mode == "RGBA":
    img = img.convert("RGB")
image = pipe(
    prompt="generate fan-shaped double eye lids and keep everything else unchanged",
    edit_image=img,
    height=height, width=width,
    seed=0,
    num_inference_steps=15,
    negative_prompt="long eye lashes"
)
image.save(f"test_{i}_ks.png")
`

diffusers代码：
`image1 = Image.open(f"test_{i}.png")
if image1.mode == "RGBA":
    image1 = image1.convert("RGB")
prompt = "generate fan-shaped double eye lids and keep everything else unchanged"
inputs = {
    "image": [image1],
    "prompt": prompt,
    "generator": torch.manual_seed(0),
    "true_cfg_scale": 4.0,
    "negative_prompt": "long eye lashes",
    "num_inference_steps": 15,
    "guidance_scale": 1.0,
    "num_images_per_prompt": 1,
}
with torch.inference_mode():
    output = pipeline(**inputs)
    output_image = output.images[0]
    output_image.save(f"test_{i}_ks_base_model_by_diffusers.png")`

差异挺大的，使用diffsynth-studio的一致性远不如diffusers，是什么原因导致的？

**Comment by Artiprocher** [comment]
@yangguoquan001 这是分辨率原因导致的，diffusers 默认会 resize 输入图片并把输出图像的分辨率设置为相同的，我们的输出分辨率必须手动设置，当 `edit_auto_resize=False` 并在外部手动对齐分辨率时，我们的输出图与 diffusers 是几乎完全一致的

**Comment by yangguoquan001** [comment]
> [@yangguoquan001](https://github.com/yangguoquan001) 这是分辨率原因导致的，diffusers 默认会 resize 输入图片并把输出图像的分辨率设置为相同的，我们的输出分辨率必须手动设置，当 `edit_auto_resize=False` 并在外部手动对齐分辨率时，我们的输出图与 diffusers 是几乎完全一致的

如何在外部手动对齐分辨率 @Artiprocher

**Comment by mzy1999** [comment]
> > [@yangguoquan001](https://github.com/yangguoquan001) 这是分辨率原因导致的，diffusers 默认会 resize 输入图片并把输出图像的分辨率设置为相同的，我们的输出分辨率必须手动设置，当 `edit_auto_resize=False` 并在外部手动对齐分辨率时，我们的输出图与 diffusers 是几乎完全一致的
> 
> 如何在外部手动对齐分辨率 [@Artiprocher](https://github.com/Artiprocher)

请问这个问题解决了吗？
